lms/static/sass/shared/_footer-edx.scss

```diff
@@ -516,7 +687,44 @@ footer#footer-edx-v3 {
   @include media-breakpoint-up(lg) {
     padding: $baseline $baseline*0.5;
 
-    .margin {
+    .column-1 {
+      flex: 0 0 100%;
+      max-width: 100%;
+    }
+
+    .column-2 {
+      flex: 0 0 66.66667%;
+      max-width: 66.66667%;
+    }
+
+    .column-3 {
+      flex: 0 0 33.3333%;
+      max-width: 33.3333%;
+    }
+
+    .column-2-1 {
+      flex: 0 0 33.3333%;
+      max-width: 33.3333%;
+    }
```

Comment:
Please update to group selectors that have the same styles applied to them (for these lines and the rest of your changes).